packages/pwa-kit-runtime/src/utils/ssr-proxying.js

```diff
@@ -525,8 +526,9 @@ export const rewriteSetCookies = ({appHostname, setCookies, targetHost, logging}
 
             /* istanbul ignore next */
             if (logging) {
-                console.log(
-                    `Rewriting proxy response set-cookie header domain from "${cookie.domain}" to "${newDomain}"`
+                logger.info(
+                    `Rewriting proxy response set-cookie header domain from "${cookie.domain}" to "${newDomain}"`,
+                    {namespace: 'ssr-proxying.rewriteProxyResponseHeaders.rewriteSetCookies'}
```

Comment:
By default the logger being used in the runtime (the one referenced from utiles/logger) has a namespace `pwa-kit-runtime`, here we are passing in a new namespace, but it doesn't include `pwa-kit-runtime` at the front. Should it?